mmcv/transforms/processing.py

```diff
@@ -1217,18 +1218,16 @@ def __repr__(self) -> str:
 class RandomResize(BaseTransform):
     """Random resize images & bbox & keypoints.
 
-    Added or updated keys: scale, scale_factor, keep_ratio, img, height, width,
-    gt_bboxes, gt_semantic_seg, and gt_keypoints.
     How to choose the target scale to resize the image will follow the rules
     below:
 
-    - if `scale` is a list of tuple, the first value of the target scale is
-      sampled from [`scale[0][0]`, `scale[1][0]`] uniformally and the second
-      value of the target scale is sampled from [`scale[0][1]`, `scale[1][1]`]
-      uniformally.
-    - if `scale` is a tuple, the first and second values of the target scale
-      is equal to the first and second values of `scale` multiplied by a value
-      sampled from [`ratio_range[0]`, `ratio_range[1]`] uniformally.
+    - if ``scale`` is a list of tuple, the first value of the target scale is
+      sampled from [``scale[0][0]``, ``scale[1][0]``] uniformally and the
+      second value of the target scale is sampled from
+      [``scale[0][1]``, ``scale[1][1]``] uniformally.
```

Comment:
might mention 'follow xx convention,scale[0][0] is for height? and  scale[1][0] for width?'